src/components/Dialog/Dialog.js

```diff
@@ -126,32 +116,21 @@ Dialog.propTypes = {
    * Determine the container for scrolling the dialog.
    */
   scroll: PropTypes.oneOf(['body', 'paper']),
-  /**
- * Enter event to use with UXPin interactions.
- */
-  onEnter: PropTypes.func,
 
-  /**
-   * Exit event to use with UXPin interactions.
-   */
-  onExit: PropTypes.func,
-  
```

Comment:
Do we not need these anymore?
For instance if we wanted another prop to update on Dialog close we could use this.

Reply:
I removed these 2 props because I couldn't find them in the doc: https://mui.com/material-ui/api/dialog

So it looked like something that cannot work but maybe I'm missing something?

Reply:
Ah yes, I was looking at the old [v4 API](https://v4.mui.com/api/dialog/) by mistake. You are correct, they have been removed for MUI v5.